build/loc.proj

```diff
@@ -27,7 +27,7 @@
         <DestinationDir>$(ArtifactsDirectory)vsixlangpack\extension.vsixlangpack</DestinationDir>
       </NonProjectFilesToMove>
     </ItemGroup>
-    <Copy SourceFiles="@(NonProjectFilesToMove)" DestinationFiles="@(NonProjectFilesToMove->'%(DestinationDir)')"/>
+    <Copy SourceFiles="@(NonProjectFilesToMove)" DestinationFiles="@(NonProjectFilesToMove->'%(DestinationDir)')" UseHardlinksIfPossible="true" />
     <ItemGroup>
```

Comment:
Is it file system hard symbolic link?
Besides "NuGet Build Pipeline" how does it affect our local build?
What happen if original is changes or get deleted after link is created? Does it reflect exactly what happen to original file? If not then it might add sometimes very hard to diagnose build issue.

Reply:
> Is it file system hard symbolic link?

yes Erick.

> Besides, "NuGet Build Pipeline" how does it affect our local build?

I executed `build.ps1` locally and it succeeded.

> What happen if original is changes or get deleted after link is created? Does it reflect exactly what happen to original file?

File system maintains a count of references to the file, the original file won't be deleted unless the references count is zero. All the hard links are pointing to the same bits; hence changes will be global.